src/router/index.js

```diff
@@ -48,8 +50,14 @@ const routes = [
 
 const router = new VueRouter({
   mode: 'history',
+  scrollBehavior,
   base: process.env.BASE_URL,
   routes
 })
 
+new Vue({
```

Comment:
This is causing Vue to get confused and a lot of errors are being output in the browser's console. I would delete this and the related import since we are booting up the Vue app from /src/main.js.


Example of errors:
![extra vue errors](https://user-images.githubusercontent.com/54147446/216791113-67e6f810-98da-4692-96be-9b97ee54dfb5.png)

Reply:
Ok, import and code have been deleted.